doc/WALLET_DESKTOP.md

```diff
@@ -104,5 +135,9 @@ If confirmed, you will need to wait for the signing process to be done.
 The wallet will get the signature information and send the transaction, after which you will be directed to the `Wallet` screen.
 
 Observations:
-- Only HTR (Hathor native token) is supported with the ledger app.
-- Time lock is disabled for transactions with ledger.
+- Only HTR (Hathor native token) is supported with the Ledger app.
+- Time lock is disabled for transactions with Ledger.
+
+## Contact Info
+
+[![Email][email.logo]][email.link]&emsp;[![Discord][discord.logo]][discord.link]&emsp;[![Telegram][telegram.logo]][telegram.link]
```

Comment:
Shouldn't this be on README.md? 

Also, the logos look distorted here. Not sure if it's only here.
![image](https://user-images.githubusercontent.com/3236756/153072175-1fd06a68-30bd-4c57-9f4d-3fe32b9663eb.png)
If it's distorted everywhere, I'd remove the images and just add:
```
Website: ...
Discord: ...
Email: ...
```

Reply:
Done in [5fb2acb](https://github.com/HathorNetwork/hathor-ledger-app/pull/50/commits/5fb2acbcaae7ae49097f1c7f6b6b569498cf8948)